### DISCUSSION TITLE
Source and License of Avalonia Fluent Icons

### DISCUSSION THREAD

**Discussion by Elbon-Eastmage** [AvaloniaUI/Avalonia#14285] [Q&A] [answered]
I came across a collection of stream geometry icons [here](https://avaloniaui.github.io/icons.html), and I'm thinking they could be very useful for me. However, there are a couple of things I'm wondering about.

1. Did these icons come from the Avalonia team?
2. What is the usage license for them?

**Comment by thevortexcloud** [comment, marked as answer]
They apparently came from Microsoft. They are also MIT licensed

https://github.com/AvaloniaUI/fluentui-system-icons/blob/master/LICENSE

**Reply by Elbon-Eastmage** [reply]
Thank you so much!
